docker/windows/base/build/Dockerfile

```diff
@@ -34,6 +34,8 @@ RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.or
     choco install -y -c C:\choco-cache nodejs-lts;`
     choco install -y -c C:\choco-cache git;`
     choco install -y -c C:\choco-cache cmake --installargs 'ADD_CMAKE_TO_PATH=System';`
+    choco install -y -c C:\choco-cache nasm;`
+    setx /M PATH $($Env:PATH + ';C:\Program Files\NASM');`
```

Comment:
this won't work since the enclosing shell is powershell:

See the line further below for an example how to set environment variables at machine level:

```
cmd /S /C setx /M PYTHONIOENCODING 'UTF-8';`
```

Reply:
We have the exact same code in the internal containers right now and it does work. Why would it be different here ?

Reply:
if it works then sgtm -- I was under the impression this was still not working like when I was first implementing this (and environment variables were not updating)